lib/super_digit.rb

```diff
@@ -1,15 +1,26 @@
 # Superdigit
 
-# Time Complexity - ?
-# Space Complexity - ?
+# Time Complexity - ? O(n)
+# Space Complexity - ? O(n)
+
+#Given an integer, we need to find the super digit of the integer.
+
 def super_digit(n)
```

Comment:
This works but `n.to_s.chars` is O(n) as is the `.map` method.  Then you call `super_digit` so this method will have a bigO bigger than O(n).